ReactiveCocoa/Swift/Property.swift

```diff
@@ -355,61 +325,61 @@ public struct AnyProperty<Value>: PropertyType {
 	}
 
 	/// Initializes a property as a read-only view of the given property.
-	public init<P: PropertyType where P.Value == Value>(_ property: P) {
-		sources = AnyProperty.capture(property)
+	public init<P: PropertyProtocol where P.Value == Value>(_ property: P) {
+		sources = AnyProperty.capturing(property)
 		disposable = nil
 		_value = { property.value }
 		_producer = { property.producer }
 		_signal = { property.signal }
 	}
 
-	/// Initializes a property that first takes on `initialValue`, then each value
+	/// Initializes a property that first takes on `initial`, then each value
 	/// sent on a signal created by `producer`.
-	public init(initialValue: Value, producer: SignalProducer<Value, NoError>) {
-		self.init(propertyProducer: producer.prefix(value: initialValue),
+	public init(initial: Value, followingBy producer: SignalProducer<Value, NoError>) {
```

Comment:
I don't think the `followingBy` here is a good name. We should stick with `producer` or find some other noun that works.